java/ql/src/Security/CWE/CWE-1104/MavenPomDependsOnBintray.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p><a href="https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/">Bintray and JCenter are shutting down on May 1st, 2021</a>.
+Relying upon repositories that are deprecated or slated to be shutdown can have unintended consequences;
+for example, artifacts being resolved from different artifact server or total breakage of the CI build.</p>
+
+<p>When artifact repositories are left unmaintained for a long period of time, vulnerabilities may emerge.
+Theoretically, this could allow attackers to inject malicious code into the artifacts that you are resolving and infect build artifacts
+that are being produced. This can be used by attackers to perform a
+<a href="https://en.wikipedia.org/wiki/Supply_chain_attack">Supply chain attack</a>
```

Comment:
```suggestion
<a href="https://en.wikipedia.org/wiki/Supply_chain_attack">supply chain attack</a>
```
Since it appears in the middle of the sentence it might make sense to lowercase it.